README.md

```diff
@@ -30,3 +31,32 @@ function mystery(n) {
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+$T(n) =$ 1 if $n <= 1$ <br>
+$3T(\frac{n}{3}) + n^5$ + C otherwise <br>
+
+Solve by substitution 
+
+$T(n)=3T(\frac{n}{3})+n^5+C$
+
+$=3[3T(\frac{n}{9})+(\frac{n}{3})^5+C]+n^5+C$
+
+$=9T(\frac{n}{9})+3(\frac{n}{3})^5+3C+n^5+C$
+
+$=9T(\frac{n}{9})+3(\frac{n}{3})^5+n^5+3C+C$
+
+$=3^kT(\frac{n}{3^k})+\sum\limits_{i=0}^{k-1} 3^i(\frac{n}{3^i}) +\sum\limits_{i=0}^{k-1} 3^iC$
```

Comment:
The first sum would just simplify to $n$, wouldn't it?

Reply:
$\sum\limits_{i=0}^{k-1} 3^i(\frac{n}{3^i})$ This?

Reply:
Yes.

Reply:
I'm confused on how it would simplify to that. Its a geometric series that would simplify to $n^5\cdot \frac{1-3^{-4k}}{1-3^{-4}}$

Reply:
Oh shoot I dont have n^5 in the numerator, is that the issue?

Reply:
The $3^i$ simply cancels out. I think you're missing something in that expression.

Reply:
I suggest expanding the expression one more step for T(n/27) to get a better guess on the summation part.